GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/ContentTypes/accordion.config

```diff
@@ -14,9 +14,7 @@
       <KeepAllVersionsNewerThanDays></KeepAllVersionsNewerThanDays>
       <KeepLatestVersionPerDayForDays></KeepLatestVersionPerDayForDays>
     </HistoryCleanup>
-    <Compositions>
-      <Composition Key="086143f0-2a9d-431a-8480-a4e634131f9b">govukPageSettingHeadingSize</Composition>
-    </Compositions>
+    <Compositions />
```

Comment:
Removing this composition from document types is why the `<headingScaleStart>` property is getting removed from a lot of pages.